code/client/clrcore-v2/Attributes.cs

```diff
@@ -116,6 +116,32 @@ public EventHandlerAttribute(string name, Binding binding = Binding.All)
 			Binding = binding;
 		}
 	}
+	
+#if !IS_FXSERVER
+	/// <summary>
+	/// Register this method to listen for the given <see cref="CallbackEvent"/> when this <see cref="BaseScript"/> is loaded
+	/// if <see cref="IsRawCallback"/> is specified this will use a raw NUI callback instead
+	/// </summary>
+	/// <remarks>Only works on <see cref="BaseScript"/> inherited class methods</remarks>
+#else
+	/// <summary>Does nothing on server side</summary>
+	[EditorBrowsable(EditorBrowsableState.Never)]
+#endif
+	[AttributeUsage(AttributeTargets.Method, AllowMultiple = true)]
+	public class NuiCallbackAttribute : Attribute
+	{
+		public string CallbackEvent { get; }
+		public bool IsRawCallback { get; }
+		public NuiCallbackAttribute(string callbackEvent, bool isRawCallback = false)
```

Comment:
`CallbackEvent` indicates that this is the event and/or callback. I'd suggest something like `Name` or what we generally call this part:
```
                                   vvvvvvvvvvv
https://${GetParentResourceName()}/getItemInfo
```

Could you also apply the same changes in the other places?